staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4911,6 +4911,7 @@ type PodPortForwardOptions struct {
 	Ports []int32 `json:"ports,omitempty" protobuf:"varint,1,rep,name=ports"`
 }
 
+// +k8s:conversion-gen:custom-from=net/url.Values
```

Comment:
I think since the target of this object is an API object, this should just be in generated.  I don't think it should be a separate file.  It's a "one way conversion" but it feels no different than normal generated to me.

Reply:
Agree.